core/src/main/antlr/xtdb/antlr/Sql.g4

```diff
@@ -266,6 +273,10 @@ exprPrimary
 
     | (schemaName '.')? 'VERSION' '(' ')' #PostgresVersionFunction
 
+    | (schemaName '.')? 'PG_GET_USERBYID' '(' relOwner=columnReference ')' #PostgresGetUserbyidFunction
```

Comment:
please make this casing consistent with the rest of the file: `PostgresGetUserByIdFunction`